lib/bedrock.js

```diff
@@ -320,6 +243,59 @@ api.exit = function() {
   cluster.worker.kill();
 };
 
+async function _runOnce({type, id, fn, options}) {
+  // listen for scheduling message from master
+  const promise = _waitForOneMessage({type, id});
+
+  // notify master to schedule work (if not already scheduled/run)
+  process.send({type, id, options});
+
+  // wait for scheduling result
+  let msg = await promise;
```

Comment:
can we just `let msg = await _waitForOneMessage(...)`?  I don't see promise being used after this line.

Reply:
@mattcollier, yeah we can do that ... I was thinking that "run to completion" rules didn't apply to `process.send` for some reason and that we needed to add a listener before notifying ... but that's just crazy. Changed it.